concent_api/core/subtask_helpers.py

```diff
@@ -311,7 +313,12 @@ def finalize_deposit_claim(
     )
 
     if deposit_claim:
```

Comment:
It's a model, not a boolean. It's not a change from your pull request but the fix is trivial so please change it to `if deposit_claim is not None:`.

Reply:
done